src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -20,6 +20,44 @@ def multi_instrument_session():
         yield simulated_session
 
 
+def test_reset(multi_instrument_session):
+    multi_instrument_session.selected_function = nidigital.SelectedFunction.PPMU
+    assert multi_instrument_session.selected_function == nidigital.SelectedFunction.PPMU
+    multi_instrument_session.reset()
+    assert multi_instrument_session.selected_function == nidigital.SelectedFunction.DISCONNECT
+
+
+def test_reset_device(multi_instrument_session):
+    multi_instrument_session.selected_function = nidigital.SelectedFunction.PPMU
+    assert multi_instrument_session.selected_function == nidigital.SelectedFunction.PPMU
+    multi_instrument_session.reset_device()
+    assert multi_instrument_session.selected_function == nidigital.SelectedFunction.DISCONNECT
+
+
+def test_self_test(multi_instrument_session):
+    multi_instrument_session.self_test()
+
+
+def test_get_error(multi_instrument_session):
```

Comment:
shouldn't this be private?

Reply:
`_get_error()` is private. This test is to verify nidigital module bundles the error code and description correctly in the exception object it throws.
Test is similar to what we [have](https://github.com/ni/nimi-python/blob/7911a1400de731e03b620ee9492599aa19f67dcc/src/nidcpower/system_tests/test_system_nidcpower.py#L54) as part of other system tests.